src/Guid.ts

```diff
@@ -97,16 +99,11 @@ export class Guid implements IEquatable, IFormattable, IJsonStringifyable {
         s = Guid.unify(s);
 
         // if the value parameter is valid
-        if (/^[0-9ABCDEF]{32}$/.test(s)) {
-            const svo = new Guid();
-            svo.v = Guid.unstrip(s);
-            return svo;
-        }
-        // return undefined if creation failed.
-        return undefined;
+        return /^[0-9ABCDEF]{32}$/.test(s)
```

Comment:
To prove my point earlier what is `s`

Reply:
`s` is the input string. Making it `str` or `input` does not change a lot. Furthermore, using `s` as the name of input string is quite common, and it not part of this PR. ;)